pkg/services/rendering/http_mode.go

```diff
@@ -83,42 +112,85 @@ func (rs *RenderingService) renderViaHttp(ctx context.Context, renderKey string,
 		}
 	}()
 
+	var downloadFileName string
+	_, params, err := mime.ParseMediaType(resp.Header.Get("Content-Disposition"))
+	if err == nil {
+		downloadFileName = params["filename"]
+	}
+
+	err = rs.readFileResponse(reqContext, resp, filePath)
+	if err != nil {
+		return nil, err
+	}
+
+	return &RenderCSVResult{FilePath: filePath, FileName: downloadFileName}, nil
+}
+
+func (rs *RenderingService) doRequest(ctx context.Context, url *url.URL, headers map[string][]string) (*http.Response, error) {
+	req, err := http.NewRequest("GET", url.String(), nil)
+	if err != nil {
+		return nil, err
+	}
+
+	req.Header.Set("User-Agent", fmt.Sprintf("Grafana/%s", rs.Cfg.BuildVersion))
+	for k, v := range headers {
+		req.Header[k] = v
+	}
+
+	req = req.WithContext(ctx)
+
+	rs.log.Debug("calling remote rendering service", "url", url)
+
+	// make request to renderer server
+	resp, err := netClient.Do(req)
+	if err != nil {
+		rs.log.Error("Failed to send request to remote rendering service.", "error", err)
```

Comment:
```suggestion
		rs.log.Error("Failed to send request to remote rendering service", "error", err)
```

It's super nitpick 🙏🏻  But that dot at the end of the log line caught my attention 😳